server/rest.py

```diff
@@ -167,3 +176,42 @@ def deleteTiles(self, item, params):
         return {
             'deleted': deleted
         }
+
+    @describeRoute(
+        Description('Get a thumbnail of a large image item.')
+        .notes('Aspect ratio is always preserved.  If both width and height '
+               'are specified, the resulting thumbnail may be smaller in one '
+               'of the two dimensions.  If neither width nor height is given, '
+               'a default size will be returned.  '
+               'This creates a thumbnail from the lowest level of the source '
+               'image, which means that asking for a large thumbnail will not '
+               'be a high-quality image.')
+        .param('itemId', 'The ID of the item.', paramType='path')
+        .param('width', 'The maximum width of the thumbnail in pixels.',
+               required=False, dataType='int')
+        .param('height', 'The maximum height of the thumbnail in pixels.',
+               required=False, dataType='int')
+        .param('encoding', 'Thumbnail output encoding', required=False,
+               enum=['PNG', 'JPEG'], default='PNG')
+        .param('jpegQuality', 'Quality used for generating JPEG images',
```

Comment:
Let's drop ``jpegQuality`` and ``jpegSubsampling`` from the thumbnail HTTP API for now. Leave them available internally in the model / tilesource.

We will definitely want to expose these in the HTTP API for a more generalized "region cutout" endpoint in the future.